Fails to build, perhaps not safe to build in several threads (COMPONENT_BUILD_ARGS=-j4).
```
/bin/bash ../../libtool --tag=F77 --mode=link /usr/gcc/4.9/bin/gfortran -I/export/home/alp/srcs/tests/oi-userland/components/image/plplot/plplot-5.8.0/bindings/f77 -m64  -m64 -o libplplotf77d.la -rpath /usr/lib/amd64 -version-info 10:0:1 -rpath /usr/lib/amd64 -no-undefined libplplotf77cd.la strutil.lo sfstubs.lo configurable.lo  -lm 
libtool: link: cannot find the library `libplplotf77cd.la' or unhandled argument `libplplotf77cd.la'
make[4]: *** [Makefile:602: libplplotf77d.la] Error 1
```

--- a/components/image/plplot/Makefile
+++ b/components/image/plplot/Makefile
@@ -1,0 +1,51 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2017 Dariusz Sendkowski
+#
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		plplot
+COMPONENT_VERSION=	5.8.0
+COMPONENT_PROJECT_URL=	http://plplot.sourceforge.net
+COMPONENT_SUMMARY=	A cross-platform software package for creating scientific plots.
+COMPONENT_FMRI=         image/plplot
+COMPONENT_CLASSIFICATION=	Applications/Graphics and Imaging
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:595bcc021b415ba191979586543f95285f03f5b54ed77f452976a89ad2850877
+COMPONENT_ARCHIVE_URL=	http://sourceforge.net/projects/$(COMPONENT_NAME)/files/$(COMPONENT_NAME)/$(COMPONENT_VERSION)%20Source/$(COMPONENT_ARCHIVE)/download
+COMPONENT_LICENSE=	LGPLv2
+COMPONENT_LICENSE_FILE=	COPYING.LIB
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+CONFIGURE_BINDIR.64=	$(CONFIGURE_BINDIR.32)
+
+COMPONENT_BUILD_ARGS = 
+
+build:		$(BUILD_64)
+
+install:	$(INSTALL_64)
+
+test:		$(TEST_64)
+
+REQUIRED_PACKAGES += shell/bash
+REQUIRED_PACKAGES += SUNWcs
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/freetype-2
+REQUIRED_PACKAGES += system/library/g++-4-runtime
+REQUIRED_PACKAGES += system/library/gcc-4-runtime
+REQUIRED_PACKAGES += system/library/gfortran-4-runtime
+REQUIRED_PACKAGES += system/library/math
+REQUIRED_PACKAGES += x11/library/libx11

--- a/components/image/plplot/manifests/sample-manifest.p5m
+++ b/components/image/plplot/manifests/sample-manifest.p5m
@@ -1,0 +1,200 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/plm2gif
+file path=usr/bin/plplot-config
+file path=usr/bin/plplot_libtool
+file path=usr/bin/plpr
+file path=usr/bin/plrender
+file path=usr/bin/pltek
+file path=usr/bin/pstex2eps
+file path=usr/include/plplot/disptab.h
+file path=usr/include/plplot/drivers.h
+file path=usr/include/plplot/gcw.h
+file path=usr/include/plplot/pdf.h
+file path=usr/include/plplot/plConfig.h
+file path=usr/include/plplot/plDevs.h
+file path=usr/include/plplot/pldebug.h
+file path=usr/include/plplot/pldll.h
+file path=usr/include/plplot/plevent.h
+file path=usr/include/plplot/plplot.h
+file path=usr/include/plplot/plplotP.h
+file path=usr/include/plplot/plplotcanvas.h
+file path=usr/include/plplot/plstream.h
+file path=usr/include/plplot/plstrm.h
+file path=usr/include/plplot/pltcl.h
+file path=usr/include/plplot/plxwd.h
+file path=usr/include/plplot/tclMatrix.h
+file path=usr/lib/$(MACH64)/libcsirocsa.a
+link path=usr/lib/$(MACH64)/libcsirocsa.so target=libcsirocsa.so.0.0.1
+link path=usr/lib/$(MACH64)/libcsirocsa.so.0 target=libcsirocsa.so.0.0.1
+file path=usr/lib/$(MACH64)/libcsirocsa.so.0.0.1
+file path=usr/lib/$(MACH64)/libplplotcxxd.a
+link path=usr/lib/$(MACH64)/libplplotcxxd.so target=libplplotcxxd.so.9.2.0
+link path=usr/lib/$(MACH64)/libplplotcxxd.so.9 target=libplplotcxxd.so.9.2.0
+file path=usr/lib/$(MACH64)/libplplotcxxd.so.9.2.0
+file path=usr/lib/$(MACH64)/libplplotd.a
+link path=usr/lib/$(MACH64)/libplplotd.so target=libplplotd.so.11.0.1
+link path=usr/lib/$(MACH64)/libplplotd.so.11 target=libplplotd.so.11.0.1
+file path=usr/lib/$(MACH64)/libplplotd.so.11.0.1
+file path=usr/lib/$(MACH64)/libplplotf77cd.a
+link path=usr/lib/$(MACH64)/libplplotf77cd.so target=libplplotf77cd.so.9.1.0
+link path=usr/lib/$(MACH64)/libplplotf77cd.so.9 target=libplplotf77cd.so.9.1.0
+file path=usr/lib/$(MACH64)/libplplotf77cd.so.9.1.0
+file path=usr/lib/$(MACH64)/libplplotf77d.a
+link path=usr/lib/$(MACH64)/libplplotf77d.so target=libplplotf77d.so.9.1.0
+link path=usr/lib/$(MACH64)/libplplotf77d.so.9 target=libplplotf77d.so.9.1.0
+file path=usr/lib/$(MACH64)/libplplotf77d.so.9.1.0
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd-c++.pc
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd-f77.pc
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd.pc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/hpgl.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/hpgl.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/mem.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/mem.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/null.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/null.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/pbm.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/pbm.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/plmeta.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/plmeta.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/ps.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/ps.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xfig.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xfig.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xwin.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xwin.so
+file path=usr/share/doc/plplot/AUTHORS
+file path=usr/share/doc/plplot/COPYING.LIB
+file path=usr/share/doc/plplot/ChangeLog
+file path=usr/share/doc/plplot/Copyright
+file path=usr/share/doc/plplot/FAQ
+file path=usr/share/doc/plplot/NEWS
+file path=usr/share/doc/plplot/PROBLEMS
+file path=usr/share/doc/plplot/README
+file path=usr/share/doc/plplot/README.1st.csa
+file path=usr/share/doc/plplot/README.csa
+file path=usr/share/doc/plplot/README.drivers
+file path=usr/share/doc/plplot/README.pkg-config
+file path=usr/share/doc/plplot/README.release
+file path=usr/share/doc/plplot/README.wxwidgets
+file path=usr/share/man/man1/plm2gif.1
+file path=usr/share/man/man1/plplot_libtool.1
+file path=usr/share/man/man1/plpr.1
+file path=usr/share/man/man1/pltek.1
+file path=usr/share/man/man1/pstex2eps.1
+file path=usr/share/plplot$(COMPONENT_VERSION)/cglobe.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/Makefile
+link path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/lena.pgm \
+    target=../c/lena.pgm
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/plc++demos.h
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x01.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x01cc.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x02.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x03.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x04.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x05.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x06.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x07.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x08.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x09.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x10.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x11.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x12.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x13.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x14.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x15.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x16.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x17.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x18.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x19.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x20.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x21.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x22.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x23.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x24.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x26.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/lena.pgm
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/plcdemos.h
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/tutor.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x01c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x02c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x03c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x04c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x05c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x06c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x07c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x08c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x09c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x10c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x11c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x12c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x13c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x14c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x15c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x16c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x17c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x18c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x19c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x20c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x21c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x22c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x23c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x24c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x25c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x26c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x01f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x02f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x03f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x04f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x05f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x06f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x07f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x08f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x09f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x10f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x11f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x12f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x13f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x14f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x15f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x16af.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x16f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x17f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x18f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x19f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x22f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x23f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/plplot-test.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_c.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_cxx.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_f77.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/globe.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/plstnd5.fnt
+file path=usr/share/plplot$(COMPONENT_VERSION)/plxtnd5.fnt
+file path=usr/share/plplot$(COMPONENT_VERSION)/usa.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/usaglobe.map

--- a/components/image/plplot/patches/01-pltclgen.patch
+++ b/components/image/plplot/patches/01-pltclgen.patch
@@ -1,0 +1,14 @@
+--- plplot-5.8.0/bindings/tcl/pltclgen	Sun Nov 18 21:04:00 2007
++++ plplot-5.8.0/bindings/tcl/pltclgen1	Wed May 10 18:35:48 2017
+@@ -25,9 +25,9 @@
+ # probably unrealistic.
+ ###############################################################################
+ 
+-require "getopts.pl";
++use Getopt::Std;
+ 
+-&Getopts('v');
++getopts('v');
+ 
+ $verbose = $opt_v;
+ 

--- a/components/image/plplot/plplot.p5m
+++ b/components/image/plplot/plplot.p5m
@@ -1,0 +1,195 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Dariusz Sendkowski
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/plm2gif
+file path=usr/bin/plplot-config
+file path=usr/bin/plplot_libtool
+file path=usr/bin/plpr
+file path=usr/bin/plrender
+file path=usr/bin/pltek
+file path=usr/bin/pstex2eps
+file path=usr/include/plplot/disptab.h
+file path=usr/include/plplot/drivers.h
+file path=usr/include/plplot/gcw.h
+file path=usr/include/plplot/pdf.h
+file path=usr/include/plplot/plConfig.h
+file path=usr/include/plplot/plDevs.h
+file path=usr/include/plplot/pldebug.h
+file path=usr/include/plplot/pldll.h
+file path=usr/include/plplot/plevent.h
+file path=usr/include/plplot/plplot.h
+file path=usr/include/plplot/plplotP.h
+file path=usr/include/plplot/plplotcanvas.h
+file path=usr/include/plplot/plstream.h
+file path=usr/include/plplot/plstrm.h
+file path=usr/include/plplot/pltcl.h
+file path=usr/include/plplot/plxwd.h
+file path=usr/include/plplot/tclMatrix.h
+link path=usr/lib/$(MACH64)/libcsirocsa.so target=libcsirocsa.so.0.0.1
+link path=usr/lib/$(MACH64)/libcsirocsa.so.0 target=libcsirocsa.so.0.0.1
+file path=usr/lib/$(MACH64)/libcsirocsa.so.0.0.1
+link path=usr/lib/$(MACH64)/libplplotcxxd.so target=libplplotcxxd.so.9.2.0
+link path=usr/lib/$(MACH64)/libplplotcxxd.so.9 target=libplplotcxxd.so.9.2.0
+file path=usr/lib/$(MACH64)/libplplotcxxd.so.9.2.0
+link path=usr/lib/$(MACH64)/libplplotd.so target=libplplotd.so.11.0.1
+link path=usr/lib/$(MACH64)/libplplotd.so.11 target=libplplotd.so.11.0.1
+file path=usr/lib/$(MACH64)/libplplotd.so.11.0.1
+link path=usr/lib/$(MACH64)/libplplotf77cd.so target=libplplotf77cd.so.9.1.0
+link path=usr/lib/$(MACH64)/libplplotf77cd.so.9 target=libplplotf77cd.so.9.1.0
+file path=usr/lib/$(MACH64)/libplplotf77cd.so.9.1.0
+link path=usr/lib/$(MACH64)/libplplotf77d.so target=libplplotf77d.so.9.1.0
+link path=usr/lib/$(MACH64)/libplplotf77d.so.9 target=libplplotf77d.so.9.1.0
+file path=usr/lib/$(MACH64)/libplplotf77d.so.9.1.0
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd-c++.pc
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd-f77.pc
+file path=usr/lib/$(MACH64)/pkgconfig/plplotd.pc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/hpgl.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/hpgl.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/mem.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/mem.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/null.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/null.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/pbm.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/pbm.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/plmeta.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/plmeta.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/ps.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/ps.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xfig.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xfig.so
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xwin.rc
+file path=usr/lib/$(MACH64)/plplot$(COMPONENT_VERSION)/driversd/xwin.so
+file path=usr/share/doc/plplot/AUTHORS
+file path=usr/share/doc/plplot/COPYING.LIB
+file path=usr/share/doc/plplot/ChangeLog
+file path=usr/share/doc/plplot/Copyright
+file path=usr/share/doc/plplot/FAQ
+file path=usr/share/doc/plplot/NEWS
+file path=usr/share/doc/plplot/PROBLEMS
+file path=usr/share/doc/plplot/README
+file path=usr/share/doc/plplot/README.1st.csa
+file path=usr/share/doc/plplot/README.csa
+file path=usr/share/doc/plplot/README.drivers
+file path=usr/share/doc/plplot/README.pkg-config
+file path=usr/share/doc/plplot/README.release
+file path=usr/share/doc/plplot/README.wxwidgets
+file path=usr/share/man/man1/plm2gif.1
+file path=usr/share/man/man1/plplot_libtool.1
+file path=usr/share/man/man1/plpr.1
+file path=usr/share/man/man1/pltek.1
+file path=usr/share/man/man1/pstex2eps.1
+file path=usr/share/plplot$(COMPONENT_VERSION)/cglobe.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/Makefile
+link path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/lena.pgm \
+    target=../c/lena.pgm
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/plc++demos.h
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x01.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x01cc.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x02.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x03.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x04.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x05.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x06.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x07.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x08.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x09.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x10.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x11.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x12.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x13.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x14.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x15.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x16.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x17.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x18.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x19.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x20.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x21.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x22.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x23.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x24.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c++/x26.cc
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/lena.pgm
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/plcdemos.h
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/tutor.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x01c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x02c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x03c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x04c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x05c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x06c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x07c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x08c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x09c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x10c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x11c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x12c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x13c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x14c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x15c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x16c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x17c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x18c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x19c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x20c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x21c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x22c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x23c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x24c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x25c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/c/x26c.c
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/Makefile
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x01f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x02f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x03f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x04f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x05f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x06f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x07f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x08f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x09f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x10f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x11f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x12f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x13f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x14f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x15f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x16af.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x16f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x17f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x18f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x19f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x22f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/f77/x23f.f
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/plplot-test.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_c.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_cxx.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/examples/test_f77.sh
+file path=usr/share/plplot$(COMPONENT_VERSION)/globe.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/plstnd5.fnt
+file path=usr/share/plplot$(COMPONENT_VERSION)/plxtnd5.fnt
+file path=usr/share/plplot$(COMPONENT_VERSION)/usa.map
+file path=usr/share/plplot$(COMPONENT_VERSION)/usaglobe.map

